core/src/main/java/com/linecorp/armeria/client/ClientBuilder.java

```diff
@@ -59,6 +59,14 @@ public ClientBuilder(String uri) {
         this(URI.create(requireNonNull(uri, "uri")));
     }
 
+    /**
+     * Creates a new {@link ClientBuilder} that builds the client that connects to
+     * the specified {@link Endpoint} with {@code scheme}.
+     */
+    public ClientBuilder(String scheme, Endpoint endpoint) {
```

Comment:
Could you also add the following variants:

- `ClientBuilder(Scheme scheme, Endpoint endpoint)`
- `ClientBuilder(SessionProtocol protocol, SerializationFormat format, Endpoint endpoint)`
  - Shortcut of `ClientBuilder(Scheme.of(format, protocol), endpoint)`
- `ClientBuilder(SessionProtocol protocol, Endpoint endpoint)`
  - Shortcut of `ClientBuilder(protocol, SerializationFormat.NONE, endpoint)`
- `ClientBuilder(String scheme, Endpoint endpoint)` could also be a shortcut of `ClientBuilder(Scheme.parse(scheme), endpoint)`.